<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calibFilterUI</class>
 <widget class="QWidget" name="calibFilterUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0,0,0,0);
background-image:url(://Service_Calib/Service_Calib/calib_menu.png);
border: 0px;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>96</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="serviceSetupDiagnosticLabel">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>10</y>
      <width>616</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">Service Panel/Calibrate/Filter calibration</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="rotofilterFrame">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>60</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/Service_Calib/Service_Calib/Filtro1.png);
background-image:url(:/transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="materialeLabel">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>55</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(/transparent.png);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string notr="true">Rh</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="adjustLeft">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,0);

background-image:url(://transparent.png);
border-image:url(://Pulsanti/Pulsanti/round_left.png);
border: 0px;</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QPushButton" name="adjustRight">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,0);

background-image:url(://transparent.png);
border-image:url(://Pulsanti/Pulsanti/round_right.png);
border: 0px;</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="positionLabel">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>170</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(/transparent.png);
color:rgb(65, 255, 243);</string>
     </property>
     <property name="text">
      <string notr="true">[68]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="testLabel">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>230</y>
       <width>296</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
border-image:url(/transparent.png);
color:rgb(255, 145, 55);</string>
     </property>
     <property name="text">
      <string notr="true">TEST RUNNING</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="playTest">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>380</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);

background-image:url(://transparent.png);
border-image:url(://Pulsanti/Pulsanti/play_test.png);
border: 0px;</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="stopTest">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>380</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);

background-image:url(://transparent.png);
border-image:url(://Pulsanti/Pulsanti/stop.png);
border: 0px;</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="materialeLabel_3">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>345</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string notr="true">TEST</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frameFilterButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>120</width>
      <height>286</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/transparent.png);
background-image:url(:/transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="button4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>96</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,0);

background-image:url(://transparent.png);
border-image:url(://Pulsanti/Pulsanti/scroll_left.png);
border: 0px;</string>
     </property>
     <property name="text">
      <string notr="true">   Cu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>96</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,0);

background-image:url(://transparent.png);
border-image:url(://Pulsanti/Pulsanti/scroll_left.png);
border: 0px;</string>
     </property>
     <property name="text">
      <string notr="true">   RH</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>96</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,0);

background-image:url(://transparent.png);
border-image:url(://Pulsanti/Pulsanti/scroll_left.png);
border: 0px;</string>
     </property>
     <property name="text">
      <string notr="true">   Al</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>96</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,0);

background-image:url(://transparent.png);
border-image:url(://Pulsanti/Pulsanti/scroll_left.png);
border: 0px;</string>
     </property>
     <property name="text">
      <string notr="true">   Ag</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameStore">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>65</y>
      <width>186</width>
      <height>186</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/transparent.png);
background-image:url(:/transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="storeButton">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>55</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,0);

background-image:url(://transparent.png);
border-image:url(://Service_Calib/Service_Calib/calib_icon.png);
border: 0px;</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="materialeLabel_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string notr="true">STORE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameErrors">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>255</y>
      <width>186</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/transparent.png);
background-image:url(:/transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/Pulsanti/Pulsanti/abort.png);
background-image:url(:/transparent.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>30</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
